.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-dist: trusty
```

Comment:
I think it's time to use a more uptodate distro. The good news is that it works!